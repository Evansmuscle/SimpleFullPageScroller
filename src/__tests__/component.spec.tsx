import React from "react";
import Example from "..";
import renderer from "react-test-renderer";

test("Component should show Hello World as text", () => {
  const component = renderer.create(<Example text={"World"} />);
  const testInstance = component.root;

  expect(testInstance.findByType(Example).props.text).toBe("World");

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
