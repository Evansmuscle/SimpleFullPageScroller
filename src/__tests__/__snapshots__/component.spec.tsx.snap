// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show Hello World as text 1`] = `
<div>
  Hello 
  World
</div>
`;
